ONSdigital/eq-survey-runner
# eq-533 Thankyou Page Improvements

### What is the context of this PR?

[Trello card](https://trello.com/c/MNFZTnns/533-0-5-thank-you-page-improvements)

Accessibility testing feedback indicated that the copy on this page was confusing to screen-readers. This PR implements their feedback.

Furthermore, I've refactored how the `alert` component works—removing the include as it was cumbersome to implement. I've implemented the new `alert` style on the thank-you and error pages, replacing the `box` component that was used previously.
### How to review
1. `npm run compile` 
2. Visit any survey and invoke errors, checking they still look like the screenshot below:

![screenshot 2016-09-15 16 20 11](https://cloud.githubusercontent.com/assets/930398/18555786/08cdd0e4-7b61-11e6-9def-a2e8c1bce2b5.png)
1. Successfully complete a survey, the thankyou page should look like this:

![screenshot 2016-09-15 16 23 06](https://cloud.githubusercontent.com/assets/930398/18555811/1ee1f2c0-7b61-11e6-95f2-106cc30ca49a.png)
1. Visit an error page:

![screenshot 2016-09-15 16 19 47](https://cloud.githubusercontent.com/assets/930398/18555879/59e78a10-7b61-11e6-893b-50ac98b28a3e.png)
### Who worked on the PR

@hamishtaplin.
